applicationset/examples/applications-sync-policies/guestbook/engineering-dev/guestbook-ui-deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
         app: guestbook-ui
     spec:
       containers:
-      - image: gcr.io/heptio-images/ks-guestbook-demo:0.2
+      - image: quay.io/argoprojlabs/argocd-e2e-container:0.2
```

Comment:
Here the `argocd-e2e-container` image is used with a 0.2 tag

Reply:
i dont think we should be changing the tag version. they were running a different tag before as part of the e2e. i would assume they were set intentionally?